.env.sample

```diff
@@ -1,5 +0,0 @@
-# These variables are required for end-to-end testing
-#BREVO_API_KEY=
```

Comment:
suggestion: BREVO_API_KEY & FEATURE_SEND_MAIL are not used anymore. Can you remove them once and for all?